include/irr/asset/IAsset.h

```diff
@@ -11,13 +11,20 @@ namespace asset
 
 class IAssetManager;
 
+//! A class managing Asset's extra metadata context
+/**
+	Sometimes there may be nedeed attaching some metadata by a Loader
```

Comment:
clarify

Reply:
I think it's clear, isn't it?
@Crisspl

Reply:
the english/grammar is not clear here

Reply:
Meta = above, higher

This is for attaching extra data to an asset that the loader retrieved, which are not ubiquitously representable by the engine. These could be but not limited to (note that categories overlap):
- Global Data about the file or scene
- IDs, names
- Default View/Projection
- Complex Animations and Hierarchies
- Physics Simulation Data
- AI data
- Lighting
- Extra material metadata

There is total flexibility here, its expected that each loader has its own base metadata class implementing the IAssetMetadata interface, with its own type enum that other loader's metadata classes derive from the base.

There's no attempt to standardise the metadata, as loaders are so diverse.

Reply:
status?

Reply:
Solved